apps/jacobi2d/main.c

```diff
@@ -73,247 +73,111 @@ WITH ACCESS OR USE OF THE SOFTWARE.
 // Porting to Ara SW environment
 // Author: Matteo Perotti, ETH Zurich, <mperotti@iis.ee.ethz.ch>
 
-#include <stdint.h>
+#include <stdio.h>
```

Comment:
Does this work? Is this needed? 

Afaik `stdio.h` define all kinds of functions, including stuff like `fprintf`. We do not support this, that is why we only include a `printf.h` header which only defines the `printf` function. 

Does this work well on Ara? I guess you are overwriting the definition of `printf` when you include `printf.h`?

Reply:
If SPIKE is not used it is just useless but not harmful: no `libc`-like functions should be inserted thanks to the compiler garbage collection and `printf.h` alias that transform `printf()` into a `printf_()`.